services/claim-backend/src/modules/claim/account.controller.ts

```diff
@@ -82,7 +83,11 @@ export class AccountController {
     @Param('account') account: string,
   ): Promise<CollectionResponse<ClaimEntity>> {
     return this.claimService.claims({
-      filter: { address: account },
+      filter: {
```

Comment:
I wouldn't add this here. It should go in the service. I don't think we need to pass this whole Query object anyway as we are hardcoding the values.

Reply:
So let's just have a simple `getClaimsFor(address: string)` method